### DISCUSSION TITLE
diffuse

### DISCUSSION THREAD

**Discussion by WurmD** [Homebrew/discussions#3825] [Casks]
👋 
In the recent past there seemed to be a Formula for https://diffuse.sourceforge.net/index.html but now it's unavailable.

Does anyone know why? Or how to get it back?
🙇

**Comment by gromgit** [comment]
```bash
commit 77435016c71dbbe9f205c0977fecffdc2a7f0098
Author: Michka Popoff <michkapopoff@gmail.com>
Date:   Wed Dec 18 23:26:46 2019 +0100

    diffuse: delete
    
    Does not support Python 3, needs pygtk which has been removed too.
```
So it was deleted *three years ago* (not recent) for requiring stuff that's no longer supported, so the formula simply can't be rebuilt on Homebrew-supported hardware at this point.

You'll either have to download and build the source yourself, or switch to a different tool. Sorry.

**Comment by charginghawk** [comment]


**Reply by gromgit** [reply]
This is ***NOT*** the `diffuse` that's the subject of this thread.

**Reply by p-linnane** [reply]
Comment deleted per the above.
